docs/ops/pooling/AdaptiveMaxPool_8.md

```diff
@@ -0,0 +1,87 @@
+## AdaptiveMaxPool<a name="AdaptiveMaxPool"></a> {#openvino_docs_ops_pooling_AdaptiveMaxPool_8}
+
+**Versioned name**: *AdaptiveMaxPool-8*
+
+**Category**: *Pooling*
+
+**Short description**: Applies max pooling with adaptive kernel size over the input.
+
+**Detailed description**: This operation calculates the output based on the 1st input and `output_size` determined by the 2nd input.
+The kernel dimensions are calculated using the following formulae for the `NCDHW` input case:
```

Comment:
formulae -> formula

Reply:
formulae is plural from formula, and I think there is more than one formula there...